From: Ondrej Holy <oholy@redhat.com>
Date: Fri, 21 Jan 2022 08:32:31 +0100
Subject: build: Remove incorrect i18n.merge_file argument

The positional argument was being silently ignored until meson 0.60.0 where
it fails with "ERROR: Function does not take positional arguments".

Related: https://github.com/mesonbuild/meson/issues/9441
Fixes: https://gitlab.gnome.org/GNOME/gvfs/-/issues/599
(cherry picked from commit 17a067b9b823a0d54e061eae45ff8e2c7e4a88d0)
---
 daemon/meson.build | 1 -
 1 file changed, 1 deletion(-)

diff --git a/daemon/meson.build b/daemon/meson.build
index c89ef40..620391c 100644
--- a/daemon/meson.build
+++ b/daemon/meson.build
@@ -366,7 +366,6 @@ if enable_admin
   )
 
   i18n.merge_file(
-    policy,
     input: policy_in,
     output: '@BASENAME@',
     po_dir: po_dir,
